registry/handlers/api_test.go

```diff
@@ -478,7 +478,7 @@ func testBlobAPI(t *testing.T, env *testEnv, args blobArgs) *testEnv {
 
 	// -----------------------------------------
 	// Do layer push with an empty body and different digest
-	uploadURLBase, uploadUUID = startPushLayer(t, env, imageName)
+	uploadURLBase, _ = startPushLayer(t, env, imageName)
```

Comment:
These seem like fine cleanups but don't obviously have anything to do with the commit they are in. I'll leave it to Stephen whether they should be split out.